sushy_oem_idrac/resources/manager/manager.py

```diff
@@ -83,6 +102,87 @@ class DellManagerExtension(oem_base.OEMResourceBase):
     def import_system_configuration_uri(self):
         return self._actions.import_system_configuration.target_uri
 
+    @property
+    def job_service_uri(self):
+        return self._links.job_service.resource_uri
+
+    @property
+    def get_jobs_uri(self):
+        return self._links.jobs.resource_uri
+
+    @property
+    def get_remote_service_api_uri(self):
+        return ('%s%s' %
+                (self._links.lc_service.resource_uri,
+                self.REMOTE_API_STATUS_URI))
+
+    def clear_job_queue(self, job_ids=['JID_CLEARALL']):
```

Comment:
Let's call this method "delete_jobs" since it can be used to selectively delete jobs in addition to clearing the queue